public/manifest.json

```diff
@@ -36,5 +35,10 @@
     "16": "icons/icon16.png",
     "48": "icons/icon48.png",
     "128": "icons/icon128.png"
-  }
+  },
+  "browser_specific_settings": {
+    "gecko": {
+      "id": "panda-wallet-bsv@example.com"
```

Comment:
What would this ideally be instead of "@example.com"? Should we register a domain for the project?